### PR TITLE
MULE-18788: Remove visitable components to resolve stats decoration

### PR DESCRIPTION
**Pull request by fsgonz**

### CODE REVIEW

**Comment by fsgonz** [comment]
--validate

File: core/src/main/java/org/mule/runtime/core/internal/util/message/MessageUtils.java [hunk 1]
```diff
@@ -503,35 +510,28 @@ private static Message toMessage(Result<?, ?> result, DataType dataType, Object
   }
 
   /**
-   * Generates an operation for value input decoration
+   * Decorates input value.
    * 
+   * @param v value to be decorated
    * @param eventCorrelationId the correlationId of the context involed
```
Comment (elrodro83) [inline core/src/main/java/org/mule/runtime/core/internal/util/message/MessageUtils.java:516]: involved

**Review by elrodro83 — CHANGES_REQUESTED** [review]

**Comment by fsgonz** [comment]
--validate

**Review by elrodro83 — APPROVED** [review]

**Comment by fsgonz** [comment]
--validate

**Comment by fsgonz** [comment]
--validate

**Comment by fsgonz** [comment]
--validate

**Comment by fsgonz** [comment]
--validate

**Pull request merged**